15_mini_project_work/3_magic_ball_game.py

```diff
@@ -19,6 +26,30 @@ def game_run():
     game_is_on = True
     while game_is_on:
         question = input(ENTER_QUESTION_PROMPT).lower()
+        question_is_not_correct = True
+        while question_is_not_correct:
+            no_letter = True
+            no_question_mark = True
+            is_question_impossible = False
+            for c in question:
```

Comment:
Extract two functions:
1. validate question
2. print errors (it might print several errors in one shot)

Reply:
Ok